source/Projectile.js

```diff
@@ -0,0 +1,23 @@
+var Pixi = require("pixi.js")
+var Reference = require("Reference.js")
+
+export default class Projectile extends Pixi.Sprite {
+    constructor(x, y) {
+        var projectileTex =
+            Pixi.Texture.fromImage(require("./images/projectile.png"))
+        super(projectileTex)
+        this.x = x
+        this.y = y
+        this.speed = 80
+        console.log("Creating projectile")
+    }
+    update(delta){
+            this.position.x += this.speed * delta
+            if (this.position.x < 0 || this.position.x > Reference.GAME_WIDTH ||
+                this.position.y < 0 || this.position.y > Reference.GAME_HEIGHT){
```

Comment:
Laziness is a wonderful trait for a programmer! But the Laziness that causes more work later is not the TRUE Laze.

Ommmmmmmmmmm...